models/posts_model.js

```diff
@@ -0,0 +1,19 @@
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const postSchema = new Schema({
+    title: {
+        type: String,
+        required: true
+    },
+    content: String,
+    owner: {
+        type: String,
+        required: true
+    },
+});
```

Comment:
Please accept this code, so the post document wouldn't have the generated `__v` property.

```suggestion
}, {
    versionKey: false,
});
```

Example Before:
```json
{
  "_id": "6744b4dea4ce2442aeddf377",
  "title": "Post 1",
  "content": "This is my first post",
  "owner": "Mevorah",
  "__v": 0
}
```

Example After:
```json
{
  "title": "Post 1",
  "content": "This is my first post",
  "owner": "Mevorah",
  "_id": "6744b55259e23afd32c6e2ac"
}
```